src/include/catalog/catalog.h

```diff
@@ -51,22 +51,14 @@ class CatalogContent {
         return relTableSchemas.contains(tableID);
     }
 
-    inline string getNodeTableName(table_id_t tableID) const {
-        assert(containNodeTable(tableID));
-        return nodeTableSchemas.at(tableID)->tableName;
-    }
-    inline string getRelTableName(table_id_t tableID) const {
-        assert(containRelTable(tableID));
-        return relTableSchemas.at(tableID)->tableName;
+    inline string getTableName(table_id_t tableID) const {
+        return getTableSchema(tableID)->tableName;
     }
 
-    inline NodeTableSchema* getNodeTableSchema(table_id_t tableID) const {
-        assert(containNodeTable(tableID));
-        return nodeTableSchemas.at(tableID).get();
-    }
-    inline RelTableSchema* getRelTableSchema(table_id_t tableID) const {
-        assert(containRelTable(tableID));
-        return relTableSchemas.at(tableID).get();
+    TableSchema* getTableSchema(table_id_t tableID) const {
```

Comment:
protect this function by adding contains check.

Also add back getNodeTableSchema() and getRelTableSchema()